config/user-rbac/basic-user-role.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: control-api:basic-user
+rules:
+- apiGroups: ["organization.appuio.io"] 
+  resources: ["organizations"]
+  verbs: ["get", "watch", "list", "patch", "edit", "create"]
+- apiGroups: ["rbac.appuio.io"] 
+  resources: ["organizations"]
+  verbs: ["watch", "list", "create"]
+- apiGroups: ["appuio.io"] 
+  resources: ["zones"]
+  verbs: ["get", "watch", "list"]
```

Comment:
we can generate the roles via kubebuilder?
e.g. https://github.com/k8up-io/k8up/blob/7e06b6cdc987c76a5c39755a8b831947869a2ca4/controllers/schedule_controller.go#L28
and maybe even consolidate to default view role? https://github.com/k8up-io/k8up/blob/7e06b6cdc987c76a5c39755a8b831947869a2ca4/config/rbac/consolidated_viewer_role.yaml#L8

Reply:
Aggregating to view might be a good idea. I'll try that!

I personally wouldn't use kubebuilder for that as it's not a role for the controller.

Reply:
Wait no. We don't actually want to give users the `view` role :sweat_smile: They shouldn't be able to see other resources such as namespaces or pods

Reply:
ah, right :)